SteamKit2/SteamKit2/Util/Utils.cs

```diff
@@ -49,115 +49,117 @@ public static EOSType GetOSType()
+                                14 => EOSType.MacOS1010, // "Yosemite"
+                                15 => EOSType.MacOS1011, // El Capitan
+                                16 => EOSType.MacOS1012, // Sierra
+                                17 => EOSType.Macos1013, // High Sierra
+                                18 => EOSType.Macos1014, // Mojave
+                                19 => EOSType.Macos1015, // Catalina
+                                20 => EOSType.MacOS11, // Big Sur
+                                21 => EOSType.MacOS12, // Monterey
+                                _ => EOSType.MacOSUnknown,
+                            };
+                        }
+                        else if ( RuntimeInformation.IsOSPlatform( OSPlatform.Linux ) )
+                        {
+                            // The specific minor versions only exist in Valve's enum for LTS versions
+                            return ver.Major switch
+                            {
+                                2 => ver.Minor switch
+                                {
+                                    2 => EOSType.Linux22,
+                                    4 => EOSType.Linux24,
+                                    6 => EOSType.Linux26,
+                                    _ => EOSType.LinuxUnknown,
+                                },
+                                3 => ver.Minor switch
+                                {
+                                    2 => EOSType.Linux32,
+                                    5 => EOSType.Linux35,
+                                    6 => EOSType.Linux36,
+                                    10 => EOSType.Linux310,
+                                    16 => EOSType.Linux316,
+                                    18 => EOSType.Linux318,
+                                    _ => EOSType.Linux3x,
+                                },
+                                4 => ver.Minor switch
+                                {
+                                    1 => EOSType.Linux41,
+                                    4 => EOSType.Linux44,
+                                    9 => EOSType.Linux49,
+                                    14 => EOSType.Linux414,
+                                    19 => EOSType.Linux419,
+                                    _ => EOSType.Linux4x,
+                                },
+                                5 => ver.Minor switch
+                                {
+                                    4 => EOSType.Linux54,
+                                    10 => EOSType.Linux510,
+                                    _ => EOSType.Linux5x,
+                                },
+                                6 => EOSType.Linux6x,
+                                7 => EOSType.Linux7x,
+                                _ => EOSType.LinuxUnknown,
+                            };
                         }
                         else

```

Comment:
I don't think this final else block is needed anymore, and could lie now (it could say FreeBSD is LinuxUnknown)

Reply:
Yeah I removed it.